src/components/Cart.js

```diff
@@ -0,0 +1,112 @@
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { useEffect, useState } from "react";
+
+export default function Cart({cart, setCart}) {
+
+    const [purchaseValue, setPurchaseValue] = useState(0);
+
+    function handleUpdateQuantity(value, key){
+        setCart((prev)=>({...prev,[key]:{...prev[key], quantity:value}}));
+    }
+
+    function handleDeleteCartItem(key){
+        setCart((prev)=>{ delete prev[key]; return {...prev};});
+    }
+    
+    useEffect(()=>{
+        var cartValue = 0;
+        if(cart && Object.keys(cart).length !== 0){
+            Object.keys(cart).map((k)=> {
+                cartValue += cart[k].price * cart[k].quantity;
+            });
+            setPurchaseValue(cartValue);
+        }
+
+    },[cart]);
+    
+    return (<div className="row">
+                {
+                    cart && Object.keys(cart).length === 0 ?<div className="col cart-items-text-display">Empty Cart</div>
+                :
+                <div className="cart-tile-container">
+                    <div className="row">
+                        <div className="col-lg-2 cart-items-text-display">Items In Your cart</div>
+                    </div>
+                    <div className="row ">
+                        <div className="col-lg-7 cart-tiles">
+                            {Object.keys(cart).map((k)=>
+                                <div className="cart-tile" key={k}>
+                                    <div className="row">
+                                        <div className="col-lg-2">
+                                            <div className="row">
+                                                <img src={cart[k].image}/>
+                                            </div>
+                                        </div>
+                                        
+                                        <div className="col-lg-9 cart-item-detail">
+                                            <div className="row cart-item-details cart-item-title">
+                                                <p>{cart[k].title}</p>
+                                            </div>
+                                            <div className="row cart-item-details cart-item-price">
+                                                <p><b>${cart[k].price}</b></p>
```

Comment:
@rahulmeghwal I think this is just showing per-item price. How could we restructure this so there can be a total price for all quantity of those SKUs?

Reply:
Have added a new field that shows the total price for a SKU which reflects quantity as well.